katello/01_overview.md

```diff
@@ -52,7 +52,7 @@ cases Red Hat. It also provides the possiblity to track the internal use of some
 It uses the same technology for installation as the Foreman by providing a scenario to the Foreman installer. The usage of this
 installer is the supported way of installation for Katello, so there is no way of adding the plugin later to an existing Foreman installation for now.
 
-Katello is also the upstream project for Red Hat Network Satellite 6 like Spacewalk was for Satellite 5.
+Katello is also the upstream project for Red Hat Network Satellite 6 and orcharhino like Spacewalk was for Satellite 5.
 
 Detailed information on the project homepage: http://www.katello.org/
```

Comment:
Can you add a bit more in a separate paragraph? So instead of just the `and orcharhino`, some description like 
```
orcharhino is a project by ATIX AG based on Katello which adds ...

For more information visit: https://docs.orcharhino.com/
```

Reply:
Thanks for your review and the presentation space for orcharhino :-).